netweaver/setup/swap_space.sls

```diff
@@ -0,0 +1,38 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+
+{% if netweaver.create_swap|default(true) %}
```

Comment:
@Simranpal, you forgot to update the code after updating the pillar example here

Reply:
Hi @arbulu89 yes the existing logic did not work as such for all cases, I have updated it to consider more conditions